app/components/placement/organisation_switcher_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= content_for(:header_content) do %>
+  <%= render(ContentHeaderComponent.new(
+    title: organisation.name,
+    actions: [govuk_link_to(
+      t(".change_organisation"),
+      change_organisation_link,
+      no_visited_state: true,
+    )],
+  )) %>
```

Comment:
Since this component just renders another component, it might be simpler to skip the `.html.erb` file altogether and just [define a `#call` method](https://viewcomponent.org/guide/templates.html#call).

For example, that's what's been done here:

https://github.com/DFE-Digital/itt-mentor-services/blob/99466916b2091d346122b94afaece52e90d63782/app/components/claim/status_tag_component.rb#L10-L12